src/VisualStudio/CSharp/Impl/Options/IntelliSenseOptionPageControl.xaml

```diff
@@ -28,14 +28,17 @@
                                   x:Name="ShowSnippets"
                                   Content="{x:Static local:IntelliSenseOptionPageStrings.Option_ShowSnippets}" />
                     </StackPanel>
-                </StackPanel>
-            </GroupBox>
-            <GroupBox x:Uid="SelectionInCompletionListGroupBox"
-                      Header="{x:Static local:IntelliSenseOptionPageStrings.Option_SelectionInCompletionList}">
```

Comment:
Looks like this now: 

![image](https://cloud.githubusercontent.com/assets/4564579/15765347/b7d57040-28e8-11e6-9b41-d5dbc76fa3ec.png)